.github/workflows/main.yaml

```
@@ -132,6 +132,11 @@ jobs:
 
   e2e_test:
     runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        PROTOCOL:
+          - WebSocket
+          - QUIC
```

```suggestion
          - websocket
          - quic
```
File name and shell env value look like all lowcase.

---

talked with @fisherxu ,  let us keep this